### DISCUSSION TITLE
Can I retrospectively retrieve hash data for traffic?

### DISCUSSION THREAD

**Discussion by PeterLangan** [merchantprotocol/theadslab#40] [User Guide Q&A] [answered]
We setup the web hook today on a clients website which has had the pixel installed and active for approx 3 weeks. Is there anyway to retrospectively retrieve the hash data for previous traffic?

**Comment by jonathonbyrdziak** [comment, marked as answer]
Yes actually. If you use the Google Sheet Integration it will retroactively export all of your audience to the google sheet.
